cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/dao/CradleOperators.java

```diff
@@ -41,11 +48,21 @@ public CradleOperators(CassandraDataMapper dataMapper, CassandraStorageSettings
 		this.cradleBookOp = dataMapper.cradleBookOperator(infoKeyspace, settings.getBooksTable());
 	}
 	
-	public BookOperators getOperators(BookId bookId) throws CradleStorageException
+	public BookOperators getOperators(BookId bookId, Function<BoundStatementBuilder, BoundStatementBuilder> readAttrs) throws CradleStorageException
 	{
 		BookOperators result = bookOps.get(bookId);
-		if (result == null)
-			throw new CradleStorageException("No operators prepared for book '"+bookId+"'");
+
+		if (result == null) {
+			logger.info("No BookOperators found for book {} in cache, trying to refresh it", bookId);
+			BookEntity entity = cradleBookOp.get(bookId.getName(), readAttrs);
+			if (entity == null) {
+				throw new CradleStorageException("No operators prepared for book '"+bookId+"'");
```

Comment:
The error is due to absent book information, not due to not prepared operators.

Reply:
Resolved, [8d758ea](https://github.com/th2-net/cradleapi/pull/85/commits/8d758ea710b501ede06ef3cf1cb5bb4a1699c435)

Reply:
No resolved.

Reply:
Up! This is not resolved!